pr_testing/test_multiple_prs.sh

```diff
@@ -1280,6 +1301,16 @@ if [ -z ${NO_POST} ] ; then
     if [ -d $LOCALRT/das_query ] ; then
       send_jenkins_artifacts $LOCALRT/das_query das_query/PR-${REPORT_H_CODE}/${BUILD_NUMBER}/PR || true
     fi
+    if [ -d $LOCALRT/profiling ]; then
+      for PROFILING_WORKFLOW in $PROFILING_WORKFLOWS;do
+        send_jenkins_artifacts $LOCALRT/profiling/${CMSSW_IB}/${ARCHITECTURE}/${PROFILING_WORKFLOW} profiling/${CMSSW_IB}/${ARCHITECTURE}/${PROFILING_WORKFLOW}/
```

Comment:
why not just upload `$LOCALRT/profiling/${CMSSW_IB}/${ARCHITECTURE}` to `profiling/${CMSSW_IB}/${ARCHITECTURE}` ? Do you have any thing else under  $LOCALRT/profiling/${CMSSW_IB}/${ARCHITECTURE} ?